app/src/main/java/de/opencyclecompass/app/android/ShowDataActivity.java

```diff
@@ -213,7 +224,7 @@ private void showData() {
     }
 
     private String roundDecimals(double d) {
```

Comment:
ja
```
public static String roundDecimals(double d) {
        return String.format("%.2f", d);
    }
```

Vorschlag: Hier die Util nutzen und in dieser dann noch den local - Parameter hinzufügen?